### PR TITLE
Us 12

### PR DESCRIPTION
**Pull request by MaaaChu**
## Description
Please explain briefly the changes you made here.

## Checklist
- [x] All manual security tests passing (Described here: https://github.com/Broom94/Klattice/wiki/Manual-Security-Testing)
- [x] All unit tests passing 
- [x] All UI tests passing
- [x] Code coverage suitable
- [x] WAVE toolbar run on new or updated screens
- [x] Branch named {feature|hotfix|task}/{US-.*}
- [x] If your pull request depends on any other, please link them
- [x] Approval from Tech lead
- [x] Approval from team member
- [x] Commit messages are meaningful
- [x] Target is green https://travis-ci.com/Broom94/Klattice/jobs/216924893 
- [x] Add `DO NOT MERGE` if you want to postpone merge (in labels)

### CODE REVIEW

File: api/db.js [hunk 1]
```diff
@@ -29,6 +40,8 @@ exports.getUser = function (username, userPassword){
     );
 });
 }
+const getUserQuery = "SELECT user_name, user_password, user_type FROM user WHERE user_name = ? AND user_password = ?";
```
Comment (weesteeky) [inline api/db.js]: not used, can be deleted

File: api/app.js [hunk 2]
```diff
@@ -99,6 +117,19 @@ app.post("/signout", function(req, res) {
   });
 });
 
+function updateRoleFamilies(rolefamaliesfn){
```
Comment (jamescrone) [inline api/app.js]: families* ?

File: api/app.js [hunk 3]
```diff
@@ -188,11 +219,12 @@ function updateBandTitles(bandTitlesfn) {
 }
 
 roles = [];
+roleFamiles = [];
```
Comment (jamescrone) [inline api/app.js]: same here

File: klattice/src/app/components/admin-role/admin-role.component.html [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+      >
+        <option value="">All</option>
+        <option
+          *ngFor="
+            let capability of data.capabilities
+              | async
+              | filterCapabilityByFamilyid: filter_family_id
+          "
+          value="{{ capability.capability_id }}"
+          >{{ capability.capability_name }}</option
```
Comment (jamescrone) [inline klattice/src/app/components/admin-role/admin-role.component.html]: please fix formatting here, split line up

File: klattice/src/app/components/admin-role/admin-role.component.html [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+      <select
+        class="form-control form-control-sm"
+        [(ngModel)]="filter_band_id"
+        name="filter_band_id"
+      >
+        <option value="">All</option>
+        <option
+          *ngFor="let band of data.bands | async"
+          value="{{ band.band_id }}"
+          >{{ band.band_name }}</option
```
Comment (jamescrone) [inline klattice/src/app/components/admin-role/admin-role.component.html]: same here

**Pull request merged**